inference-engine/thirdparty/clDNN/tests/test_cases/lrn_gpu_test.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#include <gtest/gtest.h>
+
+#include <api/data.hpp>
+#include <api/lrn.hpp>
+#include <api/input_layout.hpp>
+#include <api/memory.hpp>
+#include <api/network.hpp>
+#include <api/topology.hpp>
+
+#include <tests/test_utils/test_utils.h>
+
+using namespace cldnn;
+using namespace ::tests;
+
+TEST(lrn_fp32_gpu, basic) {
+    //  input : 1x16x1x1
+    //  Output : 1x16x1x1
+    const auto& engine = get_test_engine();
+
+    const size_t b = 1;
+    const size_t f = 16;
+    const size_t y = 1;
+    const size_t x = 1;
+
+    auto input = memory::allocate(engine, { data_types::f32, format::b_fs_yx_fsv16, { b, f, y, x } });
+    std::vector<float> inputVals(b * f * y * x);
+    std::generate(inputVals.begin(), inputVals.end(), []() {
+            static float n = 0;
+            return n++;
+        });
+
+    set_values(input, inputVals);
+
+    topology topology;
+    topology.add(input_layout("input", input.get_layout()));
+    topology.add(
+            lrn("lrn", "input", 2, 0.5f, 9.9e-05f, 1.f, cldnn::lrn_norm_region_across_channel)
+    );
```

Comment:
I suppose this would be better:
```
    topology.add(lrn("lrn", "input", 2, 0.5f, 9.9e-05f, 1.f, cldnn::lrn_norm_region_across_channel));
```
Or this:
```
    topology.add(
        lrn("lrn", "input", 2, 0.5f, 9.9e-05f, 1.f, cldnn::lrn_norm_region_across_channel)
    );
```
And it's better to define constants for these `2, 0.5f, 9.9e-05f, 1.f` values